dotnet/src/Planners/Planners.Core/Handlebars/Extensions/ParameterViewExtensions.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Planners.Handlebars.Models;
+
+namespace Microsoft.SemanticKernel.Planners.Handlebars.Extensions;
+
+internal static class ParameterViewExtensions
```

Comment:
Changing naming from `View` to `Metadata` see https://github.com/microsoft/semantic-kernel/pull/3545

Reply:
Refactored everything from `View` -> `Metadata`